apps/admin-x-activitypub/src/views/Feed/components/FeedList.tsx

```diff
@@ -34,6 +34,49 @@ const FeedList:React.FC<FeedListProps> = ({
     const loadMoreRef = useRef<HTMLDivElement | null>(null);
     const endLoadMoreRef = useRef<HTMLDivElement | null>(null);
 
+    // Group activities by their original post content
+    type GroupedActivity = { mainActivity: Activity; reposters: { actor: Activity['actor']; activity: Activity }[] };
```

Comment:
We have done this for feed(Notes), do we also want the same for Reader(inbox)?

<img width="1063" height="861" alt="image" src="https://github.com/user-attachments/assets/3ae4d64e-4da8-4f4b-9884-c8b60a9e6b01" />